mbq/client/contrib/permissions.py

```diff
@@ -302,6 +322,9 @@ def has_global_permission(self, person_id: UUIDType, scope: str) -> bool:
                 "scope": scope,
             },
         )
+        self.registrar.emit(
+            "has_global_permission_completed", person_id, scope, result=result
```

Comment:
What's the rationale on args vs kwargs here?

Reply:
result is always a kwarg in case people choose to pass earlier args as kwargs. Other than that I'm choosing the same approach as the method signatures